collections/_docs/en/005-003-inkscape-tips.md

```diff
@@ -2,63 +2,141 @@
 title: "Inkscape Tips"
 permalink: /docs/inkscape-tips/
 excerpt: ""
-last_modified_at: 2018-06-06
+last_modified_at: 2020-10-07
 toc: true
 ---
-Many of these tools do not create paths. To use them with Ink/Stitch you have to convert them to a path `Shift + Ctrl + C` and possibly modify them even more.
+## Tips for Ink/Stitch
+Ink/Stitch can only work with designs that are formatted as _paths_. However, many Inkscape tools (such as text and shapes) create _objects_ instead of paths. To use them with Ink/Stitch, you have to convert them by pressing `Shift + Ctrl + C` or selecting `Path > Object to Path` in the menu.
```

Comment:
This isn't exactly true anymore. Ink/Stitch can work with objects by now. Still users should know about the possibility to turn objects into paths - it's just not mandatory.